lms/djangoapps/teams/static/teams/js/spec/views/team_card_spec.js

```diff
@@ -0,0 +1,55 @@
+define(['jquery',
+        'underscore',
+        'teams/js/views/team_card',
+        'teams/js/models/team'],
+    function ($, _, TeamCardView, Team) {
+        describe('TeamCardView', function () {
+            var createTeamCardView, view;
+            createTeamCardView = function () {
+                var model = new Team({
+                    id: 'test-team',
+                    name: 'Test Team',
+                    is_active: true,
+                    course_id: 'test/course/id',
+                    topic_id: 'test-topic',
+                    description: 'A team for testing',
+                    last_activity_at: "2015-08-21T18:53:01.145Z",
+                    country: 'us',
```

Comment:
Probably doesn't matter for the test, but technically this code would be all caps `US`